hs_core/tests/api/rest/test_resource_science_meta.py

```diff
@@ -274,7 +273,7 @@ def test_put_scimeta_composite_resource_with_core_metadata_failure(self):
             ]
         }
         response = self.client.put(sysmeta_url, put_data, format='json')
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(response.status_code, status.HTTP_202_ACCEPTED)
```

Comment:
Looks like you got lucky, a test exists for this.  Could you update the test name... it still says failure, it's a happy path now.